source/getting-started/on-vm.rst

```diff
@@ -317,7 +345,7 @@ by adding :command:`--audio-passthrough` argument while invoking the
 
 .. _QEMU: https://www.qemu.org/
 
-.. _start_android_qcow2.sh: https://raw.githubusercontent.com/projectceladon/device-androidia-mixins/master/groups/device-specific/caas/start_android_qcow2.sh
+.. _start_civ.sh: https://github.com/projectceladon/vm_manager/raw/master/scripts/start_civ.sh
```

Comment:
I think users no longer require to download the scripts separately, we can remove this link.

Reply:
done